code/modules/admin/ipintel.dm

```diff
@@ -0,0 +1,138 @@
+/datum/ipintel
+	var/ip
+	var/intel = 0
+	var/cache = FALSE
+	var/cacheminutesago = 0
+	var/cachedate = ""
+	var/cacherealtime = 0
+
+/datum/ipintel/New()
```

Comment:
Shouldnt these procs really be part of the SS?

Reply:
I ported it like this because that's how TG does it.

Reply:
Fair enough